stdlib/ast.pyi

```diff
@@ -10,27 +10,28 @@ class _ABC(type):
     if sys.version_info >= (3, 9):
         def __init__(cls, *args: Unused) -> None: ...
 
-@deprecated("Replaced by ast.Constant; removal scheduled for Python 3.14")
-class Num(Constant, metaclass=_ABC):
-    value: int | float | complex
+if sys.version_info < (3, 14):
+    @deprecated("Replaced by ast.Constant; removal scheduled for Python 3.14")
```

Comment:
Nit: Consider changing "removal scheduled" to "removed".

Reply:
Removal scheduled is generally the language we use: https://github.com/search?q=repo%3Apython%2Ftypeshed%20%20removal%20schedule&type=code

Reply:
I think it's better if we stay consistent in how we phrase deprecations, though I'm not very opinionated if you still prefer we change this.

Reply:
Scheduling and actually removing are two different things. If something is scheduled for removal, there is still a chance that cpython developers change their mind, so it is better to state clearly which thing you mean.

Reply:
Fixed: https://github.com/python/typeshed/pull/12488/commits/4a1e871ec6bc940490dec192f001f7a8a3ebcabb

Reply:
I think we should use "scheduled" only for removals that haven't actually happened in CPython yet.